src/main/main.ts

```diff
@@ -126,4 +127,27 @@ app.whenReady().then(() => {
   ipcMain.handle('change-tray-icon', (event, icon: string) => {
     tray?.setImage(getTrayIcon(icon));
   });
+  ipcMain.handle('get-always-on-top', () => {
+    return mainWindow?.isAlwaysOnTop();
+  });
+  ipcMain.handle('set-always-on-top', (event, isAlwaysOnTop: boolean) => {
+    if (isAlwaysOnTop) {
+      mainWindow?.setAlwaysOnTop(true, 'screen-saver');
+    } else {
+      mainWindow?.setAlwaysOnTop(false);
```

Comment:
집중할 때 true로 설정하고, 대기 화면으로 자동으로 넘어가면 pomodoro 페이지에서 설정해준 것처럼 자동으로 false가 되면서 layer 순서가 원래대로 돌아오는데 뭔가 유저 입장에서 갑자기 사라지는게 어색할수도 있을것 같아서..!

https://github.com/user-attachments/assets/a68c12a1-8d0b-4c88-b1f9-f02b8220071c




```suggestion
  ipcMain.handle('set-always-on-top', (event, isAlwaysOnTop: boolean) => {
    if (isAlwaysOnTop) {
      mainWindow?.setAlwaysOnTop(true, 'screen-saver');
    } else {
      mainWindow?.focus();
      mainWindow?.setAlwaysOnTop(false);
```

false 설정하기 전에 focus 강제로 한번 하는건 어떨까?!

Reply:
이야 넘 좋구여~~ 👍👍
반영해둘게!